templates/user/contrib-list.html

```diff
@@ -0,0 +1,25 @@
+{% extends "user/base-users.html" %}
+
+{% block users_media %}
+    <style>
+        #users-table td:nth-child(2), #users-table th:nth-child(2) {
+            text-align: left;
+        }
+
+        #users-table th a {
+            color: white;
+        }
+    </style>
+{% endblock %}
+
+{% block title_ruler %}{% endblock %}
+
+{% block title_row %}
+    {% set tab = 'contribution' %}
+    {% set title = _('Contribution') %}
+    {% include "user/user-list-tabs.html" %}
+{% endblock %}
+
+{% block users_table %}
+    {% include "user/contrib-table.html" %}
+{% endblock %}
```

Comment:
Add end line before end of file
```suggestion
{% endblock %}

```